tests/components/network_sediment_transporter/test_bed_initializer.py

```diff
@@ -0,0 +1,272 @@
+"""
+Tests written by Jeff Keck and Allison Pfeiffer
+"""
+
+import numpy as np
+import pytest
+
+from landlab.components import (
+    BedParcelInitializerDepth,
+    BedParcelInitializerArea,
+    BedParcelInitializerUserD50,
+    BedParcelInitializerDischarge,
+)
+
+from landlab.components.network_sediment_transporter.bed_parcel_initializers import (
+    _determine_approx_parcel_volume,
+    calc_total_parcel_volume,
+    calc_d50_discharge,
+    calc_d50_depth,
+    calc_d50_dArea_scaling,
+)
+
+
+# Basic test: that you can call the four initializers
+def test_call_area_BPI(example_nmg2):
```

Comment:
These tests don't check anything other than the instances are callable. Can you add some assertions to these tests to check things about the returned values of the *initialize_parcels* calls?

Reply:
Hi @mcflugen - the tests of the content of the BedParcelInitializerBase are below. The only difference between the BedParcelInitializer components is the way D50 is calculated. The D50 calculation methods are tested here and in little doc string tests in the component itself. 

I added a check to assert that each component creates a DataRecord. 

Let me know if this satisfies, or if you have something else in mind that you think needs to be separately tested for each component.